core/controllers/suggestion.py

```diff
@@ -72,7 +126,7 @@ def post(self):
             return
 
         _upload_suggestion_images(
-            self.request,
+            self.normalized_payload,
```

Comment:
Should we pass the `self.normalized_payload.get('files')` here?

Reply:
Done